<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathDependentMC</class>
 <widget class="QDialog" name="PathDependentMC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="pdButtonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="PdSpinS">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10000000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>100.000000000000000</double>
   </property>
   <property name="value">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="PdSpinT">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.250000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>No. Simulations</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Volatility, Ïƒ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Interest, r</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Strike, K</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Underlying, S</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Option type</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="PdSpinK">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="PdSpinSimulations">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000000</number>
   </property>
   <property name="singleStep">
    <number>1000</number>
   </property>
   <property name="value">
    <number>100000</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TTM, T</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="PdSpinR">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.020000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="PdSpinSigma">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.200000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="PdComboOptionType">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Geometric Asian Call</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Geometric Asian Put</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Arithmetic Asian Call</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Arithmetic Asian Put</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>N-steps</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="PdSpinNSteps">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000000</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>12</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="PdSpinD">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>dividend, d</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pdButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>PathDependentMC</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pdButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>PathDependentMC</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
